.travis.yml

```diff
@@ -0,0 +1,14 @@
+sudo: false
+language: python
+python:
+  - 2.7
+
+install:
+  - pip install flake8
+  - nvm install 4
+  - npm config set spin false
```

Comment:
Installs node@4 (LTS build) and disables the goofy npm spinner which was putting `\|/-\|/-` nonsense in our Travis logs.